ADVANCED.md

```diff
@@ -11,26 +11,50 @@
 # Headless Core
 
 - [Headless Core Concepts](#headless-core-concepts)
-- [Headless Core Application](#headless-core-application)
+- [Working with the Headless Core](#working-with-the-headless-core)
+- [Headless Core Reference](#headless-core-reference)
 
 ## Headless Core Concepts
 
-The core is a separate library which can be used for any JavaScript based implementation.
+```
+                                    |
+    @elastic/react-search-ui        |   @elastic/search-ui
+                                    |
+                                    |
+          SearchProvider <--------------- SearchDriver
+              |     |               |          |
+   State /    |     |               |          | State /
+   Actions    |     |               |          | Actions
+              |     |               |          |
+        Components  |               |          |
+              |     |               |          |
+              v     v               |          v
+------------------------------------+----------------------------
+              |     |                          |
+              v     v                          v
+          Using     Headless Usage       Headless Usage outside
+     Components     in React             of React
+```
 
-> [The Search UI Core](https://github.com/elastic/search-ui/tree/master/packages/search-ui).
+The core is a separate, vanilla JS library which can be used for any JavaScript based implementation.
 
-All of the components in this library use the headless core under the hood.
+> [@elastic/search-ui](https://github.com/elastic/search-ui/tree/master/packages/search-ui)
 
-You can work directly with the core if you need more than the Components offer.
+One way to think of the Headless Core is as a search experience without a view. Instead of providing UI like search boxes
```

Comment:
Here's how I'd write this, maybe @goodroot can find a compromise

> The Headless Core implements the functionality behind a search experience, but without its own view. It provides the underlying "state" and "actions" associated with that UI. For instance, the core provides a `setSearchTerm` action, which can be used to save a `searchTerm` property in the state. Calling the action using the value of an `<input>` will save the `searchTerm` to be used to build a query.

Reply:
Used it with just a few small tweaks.